src/index.js

```diff
@@ -1,4 +1,89 @@
-// $(document).ready(function(){
+$(document).ready(function(){
+//selecting html elements for use in color change functions
+document.getElementById("name").onkeydown = function() {nameInputBackgroundChange()};
+document.getElementById("mail").onkeydown = function() {mailInputBackgroundChange()};
+document.getElementById("subject").onkeydown = function() {subjectInputBackgroundChange()};
+document.getElementById("msg").onkeydown = function() {msgInputBackgroundChange()};
+
+//selecting html elements for use in reverting colors back to original color upon focus change
+document.getElementById("name").onchange = function() {nameInputChange()};
+document.getElementById("mail").onchange = function() {mailInputChange()};
+document.getElementById("subject").onchange = function() {subjectInputChange()};
+document.getElementById("msg").onchange = function() {msgInputChange()};
+
+// functions to change input/text area color and background color based on userinput
+function nameInputBackgroundChange() {
+  var nameField = document.getElementById("name");
+  nameField.style.backgroundColor = "white";
```

Comment:
This is vanilla javascript, not jQuery. Let's stay consistent and write this entire file in jQuery!